File: snippets/ruby/make_your_first_api_call.rb
require 'MailchimpTransactional'

begin
  client = MailchimpTransactional::Client.new(ENV[‘YOUR_API_KEY’])
  result = client.users.ping() # => 'PONG!'
  p result
rescue MailchimpTransactional::ApiError => e
  puts "Error: #{e}"
end

File: snippets/php/make_your_first_api_call.php
<?php
require_once('/path/to/MailchimpTransactional/vendor/autoload.php');

function run(){
 try {
    $mailchimp = new MailchimpTransactional\ApiClient();
    $mailchimp->setApiKey('YOUR_API_KEY');
    $response = $mailchimp->users->ping();
    print_r($response);
  } catch (Error $e) {
        echo 'Error: ',  $e->getMessage(), "\n";
  }
}
run();